orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -248,7 +252,7 @@ def OnChange(changeType, level, resource):  # noqa: ARG001
 
 def OnHeartBeat(output, uri, **request) -> Any:  # noqa: ARG001
     """Extends the REST API by registering a new route in the REST API"""
-    orthanc.LogWarning("OK")
+    orthanc.LogInfo("OK")
```

Comment:
Agreed that this makes more sense as Info. No point in polluting the logs with a sequence of "OK"'s.